package.json

```diff
@@ -217,15 +217,35 @@
         ],
         "visible": true
       },
+      {
+        "permissionName": "ui-circulation.settings.view-overdue-fines-policies",
+        "displayName": "Settings (Circ): Can view overdue fine policies",
+        "subPermissions": [
+          "overdue-fines-policies.collection.get",
+          "overdue-fines-policies.item.get",
+          "users.collection.get",
+          "users.item.get",
+          "settings.circulation.enabled"
+        ],
+        "visible": true
+      },
       {
         "permissionName": "ui-circulation.settings.overdue-fines-policies",
         "displayName": "Settings (Circ): Can create, edit and remove overdue fine policies",
         "subPermissions": [
+          "ui-circulation.settings.view-overdue-fines-policies",
           "overdue-fines-policies.item.post",
           "overdue-fines-policies.item.put",
-          "overdue-fines-policies.item.delete",
```

Comment:
Could you please confirm that we need delete "overdue-fines-policies.item.delete" permissions here?